### PR TITLE
klient: migrate dev->prod a klient connected to prod kontrol

### PR DESCRIPTION
**Pull request by rjeczalik**

### CODE REVIEW

**Comment by rjeczalik** [comment]
@leeola @cihangir gentle ping

File: go/src/koding/klient/app/klient.go [hunk 1]
```diff
@@ -651,16 +652,14 @@ func newKite(kconf *KlientConfig) *kite.Kite {
...
+	// klient to overwrite it during installation to a
+	// proper value.
 	//
-	// Ensure that any default value is overwritten with default kontrol.
-	//
-	// TODO(rjeczalik): ensure kontrolUrl is set to a proper value.
+	// If the url was not overwritten, we default to production kontrol.
 	if k.Config.KontrolURL == "http://127.0.0.1:3000/kite" {
-		k.Config.KontrolURL = "https://koding.com/kontrol/kite"
+		k.Config.KontrolURL = "https://koding.com/kontrol/kite" // move to koding/config
```
Comment (leeola) [inline go/src/koding/klient/app/klient.go]: nitpick: Should this be a `TODO():`?

**Comment by leeola** [comment]
LGTM

**Comment by rjeczalik** [comment]
@gokmen @usirin woohooo what happened here? 💥 💣 ⚡

**Comment by gokmen** [comment]
@rjeczalik I've no idea! but it looks like this:

**Pull request closed without merging**